images/aws-ebs-csi-driver/tests/e2e/main.tf

```diff
@@ -0,0 +1,204 @@
+terraform {
+  required_providers {
+    oci        = { source = "chainguard-dev/oci" }
+    helm       = { source = "hashicorp/helm" }
+    kubernetes = { source = "hashicorp/kubernetes" }
+  }
+}
+
+variable "iam_role_arn" {}
+
+variable "kubernetes_namespace" {
+  default = "kube-system"
+}
+
+variable "kubernetes_service_account" {
+  default = "ebs-csi-controller-sa"
+}
+
+resource "kubernetes_service_account_v1" "irsa" {
+  metadata {
+    name      = var.kubernetes_service_account
+    namespace = var.kubernetes_namespace
+    annotations = {
+      "eks.amazonaws.com/role-arn" = var.iam_role_arn
+    }
+  }
+  automount_service_account_token = true
+}
+
+resource "helm_release" "ebs" {
+  name      = "aws-ebs-csi-driver"
+  namespace = var.kubernetes_namespace
+
+  repository = "https://kubernetes-sigs.github.io/aws-ebs-csi-driver"
+  chart      = "aws-ebs-csi-driver"
+
+  # TODO:
+  # version    = var.ebs_csi_driver_version
+
+  timeout = 60
+
+  values = [
+    jsonencode({
+      controller = {
+        serviceAccount = {
+          create = false
+          name   = kubernetes_service_account_v1.irsa.metadata[0].name
+        }
+      }
+    }),
+
+    # Chainguardification
+    jsonencode({
```

Comment:
leaving this in to demonstrate the horrible hack needed without access to the main tf outputs